src/pages/learn/Learn.js

```diff
@@ -0,0 +1,140 @@
+  const glossaryItems = [
+    {
+      name: "chloride",
+      image: ingredient,
+      foundIn: "Night Cream",
+      description:
+        "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore",
+    },
+    {
+      name: "isopropyl",
+      image: ingredient,
+      foundIn: "Night Cream",
+      description:
+        "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore",
+    },
+    {
+      name: "fatty accid",
+      image: ingredient,
+      foundIn: "Night Cream",
+      description:
+        "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore",
+    },
+    {
+      name: "petrolatum",
+      image: ingredient,
+      foundIn: "Night Cream",
+      description:
+        "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore",
+    },
+    {
+      name: "Oil",
+      image: ingredient,
+      foundIn: "Night Cream",
+      description:
+        "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore",
+    },
+    {
+      name: "Aloe Vera",
+      image: ingredient,
+      foundIn: "Night Cream",
+      description:
+        "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore",
+    },
+    {
+      name: "Glycerin",
+      image: ingredient,
+      foundIn: "Night Cream",
+      description:
+        "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore",
+    },
+    {
+      name: "Water",
+      image: ingredient,
+      foundIn: "Night Cream",
+      description:
+        "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore",
+    },
+    {
+      name: "Silicate",
+      image: ingredient,
+      foundIn: "Night Cream",
+      description:
+        "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore",
+    },
+  ];
+
+  const handleItemClick = (item) => {
+    setSelectedItem(item);
+  };
+
+  return (
+    <div className="Learn">
+      <header className="Learn-header">
+        <p>INGREDIENTS</p>
+      </header>
+
+      {/* Glossary Section */}
+      <div className="Glossary-container">
+        <div className="Glossary-content">
+          <ul className="Glossary-list">
+            {glossaryItems.map((item) => (
```

Comment:
this is great!! super clean way to iterate over a list so that if the list were to be altered/edited, this would update cleanly.